src/DIRAC/Resources/Computing/CloudComputingElement.py

```diff
@@ -375,37 +360,6 @@ def _getMetadata(self, executableFile, pilotStamp=""):
         userData.attach(mimeText)
         return str(userData)
 
-    def _renewCloudProxy(self):
-        """Takes short lived proxy from the site director and
-        promotes it to a long lived proxy keeping the DIRAC group.
-
-        :returns: True on success, false otherwise.
-        :rtype: bool
-        """
-        if not self._cloudDN or not self._cloudGroup:
-            self.log.error("Could not renew cloud proxy, DN and/or Group not set.")
-            return False
-
-        proxyLifetime = int(self.ceParameters.get("Context_ProxyLifetime", DEF_PROXYLIFETIME))
-        # only renew proxy if lifetime is less than configured lifetime
-        # self.valid is a datetime
-        if self.valid - datetime.datetime.utcnow() > proxyLifetime * datetime.timedelta(seconds=1):
-            return True
-        proxyLifetime += DEF_PROXYGRACE
-        proxyManager = ProxyManagerClient()
-        self.log.info(f"Downloading proxy with cloudDN and cloudGroup: {self._cloudDN}, {self._cloudGroup}")
-        res = proxyManager.downloadProxy(self._cloudDN, self._cloudGroup, limited=True, requiredTimeLeft=proxyLifetime)
-        if not res["OK"]:
-            self.log.error("Could not download proxy", res["Message"])
-            return False
-        resdump = res["Value"].dumpAllToString()
-        if not resdump["OK"]:
-            self.log.error("Failed to dump proxy to string", resdump["Message"])
-            return False
-        self.proxy = resdump["Value"]
-        self.valid = datetime.datetime.utcnow() + proxyLifetime * datetime.timedelta(seconds=1)
-        return True
-
     def __init__(self, *args, **kwargs):
```

Comment:
I think you can remove `self.proxy = ""` in `__init__()`  because it is already defined similarly in `ComputingElement`, and thus, inherited to `CloudComputingElement`.

Reply:
Thanks, yes... I'm not sure why we're initialising those there in the first place.